docker-compose.yml

```diff
@@ -114,10 +114,31 @@ services:
     build:
       context: .
       dockerfile: cpp/Dockerfile
-    environment:
-      PARQUET_TEST_DATA: /arrow/cpp/submodules/parquet-testing/data
     volumes: *ubuntu-volumes
 
+  cpp-ci:
```

Comment:
I'd use a more descriptive name which highlights that we're building against system dependencies.